NPM package capi-insights doesn't exist

It looks like one of the packages that this project depends on got taken down

## Relevant Log Portion (`make ensure-bootstrap`)
```
Step 8/23 : RUN npm install
 ---> Running in f23971b5aa27
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/sync.js'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/call-of-duty-api-84cc1453/config.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/changelog.md'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/README.md'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/axios-cdfabe32/UPGRADE_GUIDE.md'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/dgram.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/dns.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/domain.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/npm-normalize-package-bin-12b2d576/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/events.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/npm-normalize-package-bin-12b2d576/README.md'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_slash_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/incorrect_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/quux'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/same_names/foo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/without_basedir'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/mug.js'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/other_path'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/baz/quux.js'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/other_path'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/nested_symlinks/mylib/sync.js'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/module_dir/zmodules/bbb/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/baz/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/browser_field/package.json'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_slash_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/incorrect_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/invalid_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/nested_symlinks/mylib/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/symlinked/package/package.json'
npm WARN fetcher@1.0.0 No description
npm WARN fetcher@1.0.0 No repository field.

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/capi-insights/-/capi-insights-1.0.1.tgz
npm ERR! 404
npm ERR! 404  'capi-insights@1.0.1' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'fetcher'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-02T18_23_48_616Z-debug.log
The command '/bin/sh -c npm install' returned a non-zero code: 1
Makefile:78: recipe for target 'docker-build-quiet' failed
make: *** [docker-build-quiet] Error 1
```

## Full Log
```
joe@wingnut:/srv/cod-stats$ make ensure-bootstrap
Sending build context to Docker daemon  971.3kB
Step 1/23 : FROM alpine:latest
 ---> a24bb4013296
Step 2/23 : RUN apk add --no-cache --update --upgrade coreutils bash npm nodejs sqlite jq aws-cli rsync     && rm -rf /var/cache/apk/*     && mkdir /opt/app
 ---> Using cache
 ---> 0a2433a8298c
Step 3/23 : WORKDIR /opt/app
 ---> Using cache
 ---> ae87c529066d
Step 4/23 : ENV COD_DATADIR=/opt/data
 ---> Using cache
 ---> 3649cf93e3ed
Step 5/23 : VOLUME /opt/data
 ---> Using cache
 ---> 842e9e2917a1
Step 6/23 : WORKDIR /opt/app/fetcher
 ---> Using cache
 ---> 54d658031263
Step 7/23 : COPY fetcher/package.json fetcher/package-lock.json ./
 ---> Using cache
 ---> 7786e87e722d
Step 8/23 : RUN npm install
 ---> Running in f23971b5aa27
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/sync.js'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/call-of-duty-api-84cc1453/config.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/changelog.md'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/glob-5274c09f/README.md'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/axios-cdfabe32/UPGRADE_GUIDE.md'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/dgram.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/dns.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/domain.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/npm-normalize-package-bin-12b2d576/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/@types/node-d582299e/events.d.ts'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/npm-normalize-package-bin-12b2d576/README.md'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_slash_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/incorrect_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/quux'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/same_names/foo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/without_basedir'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/mug.js'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/other_path'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/baz/quux.js'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/other_path'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/nested_symlinks/mylib/sync.js'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/module_dir/zmodules/bbb/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/baz/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/browser_field/package.json'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/dot_slash_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/incorrect_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/invalid_main'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, lstat '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/multirepo'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/nested_symlinks/mylib/package.json'
npm WARN tar ENOENT: no such file or directory, open '/opt/app/fetcher/node_modules/.staging/resolve-7a0a65ba/test/resolver/symlinked/package/package.json'
npm WARN fetcher@1.0.0 No description
npm WARN fetcher@1.0.0 No repository field.

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/capi-insights/-/capi-insights-1.0.1.tgz
npm ERR! 404
npm ERR! 404  'capi-insights@1.0.1' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'fetcher'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-02T18_23_48_616Z-debug.log
The command '/bin/sh -c npm install' returned a non-zero code: 1
Makefile:78: recipe for target 'docker-build-quiet' failed
make: *** [docker-build-quiet] Error 1
```

--- a/fetcher/package.json
+++ b/fetcher/package.json
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "call-of-duty-api": "^1.2.15"
+    "call-of-duty-api": "^1.2.16"
   },
   "devDependencies": {
     "@types/node": "^14.0.13",

